packages/networking/lib/agent.ts

```diff
@@ -29,13 +43,19 @@ function createProxySock (proxy: url.Url) {
   throw new Error(`Unsupported proxy protocol: ${proxy.protocol}`)
 }
 
-function isRequestHttps(options: http.RequestOptions) {
+export function _isRequestHttps(options: http.RequestOptions) {
```

Comment:
no need to prefix this with an underscore. 

it's not really private because it's being exported and it doesn't use state or reach into other objects. it's pure and a regular ol' function up for grabs by anyone who wants it